cmd/up/main.go

```diff
@@ -176,7 +176,8 @@ func main() {
 func write(ctx context.Context, l log.Logger, opts options.Options) error {
 	switch opts.EndpointType {
 	case options.MetricsEndpointType:
-		return metrics.Write(ctx, opts.WriteEndpoint, opts.Token, metrics.Generate(opts.Labels), l, opts.TLS)
+		return metrics.Write(ctx, opts.WriteEndpoint, opts.Token, metrics.Generate(opts.Labels), l, opts.TLS,
+			opts.RemoteTenantHeader, opts.RemoteTenant)
```

Comment:
We should update this as well
```suggestion
			opts.TenantHeader, opts.Tenant)
```